cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -2018,6 +2018,37 @@ def create_formatted_datetime_string_columns(df):
     return df
 
 
+def transform_from_lookups(
+    df: DataFrame, cohort_lookup: DataFrame, travel_countries_lookup: DataFrame, tenure_group: DataFrame
+):
+    cohort_lookup = cohort_lookup.withColumnRenamed("participant_id", "cohort_participant_id")
+    df = df.join(
+        F.broadcast(cohort_lookup),
+        how="left",
+        on=((df.participant_id == cohort_lookup.cohort_participant_id) & (df.study_cohort == cohort_lookup.old_cohort)),
+    )
+    df = df.withColumn("study_cohort", F.coalesce(F.col("new_cohort"), F.col("study_cohort"))).drop(
+        "new_cohort", "old_cohort"
```

Comment:
Please could we also drop "cohort_participant_id" here? It was useful for spotting this bug, but we don't need it on the output df